src/styles/mixins/buttons.scss

```diff
@@ -1,10 +1,66 @@
-@import './focus';
+@use "./focus";
+@use "../variables" as var;
 
 @mixin reset-native-btn-styles {
-  @include outline-on-keyboard-focus;
+  @include focus.outline-on-keyboard-focus;
 
   padding: 0px;
   margin: 0px;
   background-color: transparent;
   border-style: none;
 }
+
+/**
+ * A <button> composed of an SVG icon (left) and text (right) with some
+ * hover transition effects
+ */
+@mixin action-button {
+  @include reset-native-btn-styles;
+  display: flex;
+  align-items: center;
+  padding: 0.5em;
+  border-radius: 2px;
+  border: none;
+  background-color: var.$grey-1;
+  color: var.$grey-5;
+  font-weight: 700;
+
+  &__icon {
+    color: var.$grey-5;
+    margin: 0 5px;
+  }
+
+  &:hover {
+    transition: 0.2s ease-out;
+    background-color: var.$grey-2;
+    color: var.$grey-6;
+  }
+}
+
+/**
+ * An action button with a tight icon margin
+ */
+@mixin action-button--compact {
```

Comment:
instead of a new mixin here, let's make action-button take a margin-right param and the default can be 5px

https://sass-lang.com/documentation/at-rules/mixin#arguments

Reply:
Great feedback!